export async function handleError({ error, event }) {
  console.error(error, { event })
}